docs/projects/reviews/2024/2024-sd-jwt-kotlin-annual.md

```diff
@@ -0,0 +1,39 @@
+# 2024 sd-jwt-kotlin (SD-JWT Implementation)
+
+## Project Health
+
+The main focus of this project was to provide a Kotlin implementation for the
+IETF SD-JWT draft. However, the project remain without any updates for a while and
+the maintainer decided to post an announcement stating this project is no longer maintained.
+However during TAC meeting for annual review, there was a volunteer stepped up to 
+take care of this project.
+
+
+## Maintainer Diversity
+
+As before, the project is maintained by a single maintainer.
+It's expected to have a single maintainer for the next year as well.
+
+## Project Adoption
+
+This project was used by German Bundesdruckerei but not sure they are still using.
+
+## Goals
+
+### Performance Against Prior Goals
+
+The project implemented with success to it's initial goal but not well maintained or used.
+
+### Next Year's Goals
+
+This project aims to have a cloned copy of eudi-reference-wallet-sdjwt to have every language
+implementation stays under OWF umbrella.
```

Comment:
Based on the discussion in the July 10th TAC meeting, it is important to note whether this will be a fork or a mirror. If it is a fork, how will it be different? If it is a mirror, is there a need for it or should we only have the redirect in the README?

Reply:
Mirror to the EUDI repo